ipi_tests/regression_tests/tests/NEB/DAMPED_BFGS/zundel/h5o2.dms4B.coeff.com.dat

```diff
@@ -0,0 +1,993 @@
+          dc
```

Comment:
Please use symlinks instead of copying these files. I write it only here but it applies to all similar files.